src/main.ml

```diff
@@ -186,12 +187,19 @@ let bootstrap () =
       | Error msg -> raise (Arg.Bad msg)
       | Ok c -> concurrency := Some c
     in
+    let build_logs = ref None in
+    let build_logs_arg x =
```

Comment:
We don't support polling mode in the bootstrap executable, so we shouldn't accept this option either. You can just hard-code the default value.

Reply:
Done